index.html

```diff
@@ -0,0 +1,336 @@
+			<div class="thumbnails-imgs actual" onclick="ShowSlide(idx=4)"><img src="https://images.unsplash.com/photo-1517483141275-296bb15890b2?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=f6fb898eac03ab9abd1137fa691ebe82&auto=format&fit=crop&w=1051&q=80"></div>
+			<div class="thumbnails-imgs actual" onclick="ShowSlide(idx=5)"><img src="https://images.unsplash.com/photo-1480499484268-a85a2414da81?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=6d765c5039bd6d36022739e3a6552c44&auto=format&fit=crop&w=1055&q=80"></div>
+		</div>
+		<div class="Option-bar">
+			<div class="btn info" onclick="showInfo()"></div>
+			<div class="thumbnails btn" onclick="showThumbnails()"></div>
+			<div class="counter">0/0</div>
+			<div class="btn prev" onclick="plusSlide(-1)"></div>
+			<div id="playBtn" class="btn play" onclick="play=!play; playSlides()"></div>
+			<div class="btn next" onclick="plusSlide(1)"></div>
+		</div>		
+	</div>
+<script>
+
+	//Elements
+	const imgs = document.querySelectorAll('.imgs');
+	const container = document.querySelector('.slideShow-container');
+	const thumbnailImgs = document.querySelectorAll('.thumbnails-imgs');
+	const counter = document.querySelector('.counter');
+	const popUp = document.querySelector('.thumbnails-popUp');
+	const infoBoxes = document.querySelectorAll('.paragraph');
+	const btnPlay = document.querySelector('#playBtn');
+	const refereces = document.querySelectorAll('.reference');
+	const titlesH3 = document.querySelectorAll('h3');
+
+	let intervalSlides;
+	let idx = 1;
+	let up = false;
+	let infoUp = false;
+	let refUp = false;
+	let play = false;
+
+	playSlides();
+	
+	//Function
+	function plusSlide(n) {
+		ShowSlide(idx += n);
+	}
+
+	function ShowSlide(n) {
+		var i;
+		if(n > imgs.length) {idx = 1};
+		if(n < 1){idx = imgs.length};
+		imgs.forEach(img=>img.style.display = "none");
+		thumbnailImgs.forEach(tImg=>tImg.classList.remove("actual"));
+		imgs[idx-1].style.display = "block";
+		thumbnailImgs[idx-1].classList.add("actual");
+		counter.textContent = `${idx}/${imgs.length}`;
+	}
+
+	function showThumbnails() {
+		up = !up;
+		up===true ? popUp.style.display = 'block' : popUp.style.display = 'none';
+	}
+	function showInfo() {
+		infoUp = !infoUp;
+		if(infoUp){
+			infoBoxes.forEach(infoBox => infoBox.style.display = 'block');	
+		}else{ 
+			infoBoxes.forEach(infoBox => infoBox.style.display = 'none');
+		}
+	}
+
+	function playSlides() {
+		if(play){
+			btnPlay.classList.replace("play", "pause");
+			ShowSlide(idx+=1);
+			intervalSlides = setTimeout(playSlides, 2000);
+		}else{
+			btnPlay.classList.replace("pause","play");
+			ShowSlide(idx);
+			clearTimeout(intervalSlides);
+		}
+			
+	}
+
+	function playingSlides() {
+		console.log("Hola!!");
+	}
+
+	function showReference() {
```

Comment:
You should be really careful with this. If your function is named showReference, I don't spec a variable be changed. Maybe you need an intermediary function called `changeReference` to avoid mutation in a function that should be only UI related. (this feedback applies to all the `show...` functions)

Reply:
**More Info**